rdrf/rdrf/templates/registration/registration_login_details.html

```diff
@@ -56,8 +80,9 @@
 
         {% if preferred_languages %}
             <div class="col-12">
-              <label for="id_preferred_languages">{{form.preferred_languages.label}}</label>
-              {{form.preferred_languages}}
+                <label for="id_preferred_languages">{{form.preferred_languages.label}}</label>
+                <select name="preferred_languages" class="form-control show-tick" id="id_preferred_languages"
```

Comment:
The way it moves the `input` is not great:

![image](https://github.com/eresearchqut/trrf/assets/17246877/d523d62b-4493-42a5-b6af-a8c231dae5c2)

May be enough to reduce the number of options displayed at once.